ruleset.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8" ?>
```

Comment:
Can we get rid of this file or have specific rules than general categories? Please check this PR 

https://github.com/trailheadapps/apex-recipes/pull/146/files#diff-a6839fbfcf5024903ed122d22112e9e54011183ccc884da9d1e3a3c2a9d9e04c